public/tradingview-chart.css

```diff
@@ -0,0 +1,104 @@
+.theme-dark:root {
```

Comment:
Is it our custom styles? Can we put it into TVChartContainer.scss?
It's not obvious that we have some editable files in /public folder

Reply:
The docs says the URL needs to be absolute or relative path to static/public folder. The chart is created using the IFrame element.